README.md

```diff
@@ -190,6 +190,7 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
 
   * https://www.simplybuilt.com - SimplyBuilt offers free website building and hosting for open source projects (http://www.simplybuilt.com/explore/free-websites-for-open-source-projects). Simple alternative to GitHub Pages.
   * http://www.devport.co - Turn GitHub projects, Apps, and websites into a personal developer portfolio.
+  * http://hostinger.com - PHP hosting, MySql and Site Builder, for free and no banner or ads.
```

Comment:
I am not sure this one really fits, it's not very developer focussed by the looks